lib/util.c

```diff
@@ -2099,6 +2099,8 @@ EXPORTED void xsyslog_fn(int priority, const char *description,
 {
     static struct buf buf = BUF_INITIALIZER;
     int saved_errno = errno;
+    int want_diag = (LOG_PRI(priority) != LOG_NOTICE
+                     && LOG_PRI(priority) != LOG_INFO);
```

Comment:
Nit: I think this is a rare occasion where the combining operator of a multi-line expression is put at the second line. I did a quick, non-exhaustive scan of the codebase and I only saw it a the first line (inluding util.c)

Reply:
If you want me to change it I will; but, I think this is the best way to do it, and while we're thinking about code style, it should be our style to do it this way, even though it hasn't consistently been so in the past.

Having the operator on the left, where it's aligned and easiest to see, makes the relationship between the surrounding lines a bit clearer at a glance, which saves some mental cycles for parsing the subexpressions.

For whatever it's worth, `ack '^\s*[&|]{2}'` finds hundreds of examples like this already present.  Though `ack '[&|]{2}\s*$'` finds thousands the other way, too!

Reply:
I am used to the other notation, but I am not invested in either. I'd say leave as you prefer and we address this in our upcoming style guide.